data/members/Mostafa.json

```
@@ -0,0 +1,5 @@
+{
+  "name":"Mostafa"
+  "github":"papereat"
```

```suggestion
  "name":"Mostafa",
  "github":"papereat",
```
make sure lines are separated with commas